content/en/docs/Installation/_index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Installation"
+linkTitle: "Installation"
+weight: 2
+description: > 
+    This page tells you how to get started with OLM, including installation
+---
+
+# How do I install OLM?
+
+The OLM is installable on Kubernetes clusters. For the following instructions to work, you must have a Kubernetes cluster running and the `kubectl` is able to communicate with the API server of that cluster. For more information about configuring `kubectl`, please visit [here](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/).
```

Comment:
Nit: We don't refer to OLM as The OLM, just OLM is more audibly familiar. 
Also, we can just start with the pre-req, I doubt anybody will read this info/ if this info is useful for this doc at all.
Could we stick to the bare minimums?

Reply:
sure, I have removed the content